modules/rostests/winetests/advpack/CMakeLists.txt

```diff
@@ -1,6 +1,10 @@
 
 add_definitions(-DUSE_WINE_TODOS)
 
+if(CMAKE_COMPILER_IS_GNUCC AND GCC_VERSION VERSION_GREATER 7)
```

Comment:
Hmm, we don't seem to use `CMAKE_COMPILER_IS_GNUCC` anywhere else. I guess the alternative is `NOT MSVC` (which I agree isn't better)?

Reply:
Looks like the "right" thing to do according to cmake docs, is to use `CMAKE_C_COMPILER_ID` (https://cmake.org/cmake/help/v3.2/variable/CMAKE_LANG_COMPILER_ID.html)
I'd rather use that because it is only for GCC, and won't touch clang, for example

Reply:
Was there a reason why we didn't use it?

Reply:
We use `if(GCC)` in https://github.com/reactos/reactos/blob/master/modules/rostests/apitests/crt/CMakeLists.txt